airbyte-integrations/connectors/source-auth0/Dockerfile

```diff
@@ -34,5 +34,5 @@ COPY source_auth0 ./source_auth0
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.3.0
+LABEL io.airbyte.version=1.0.0
```

Comment:
Can you change the version to a minor release instead of a major release? Major releases require a migrations file and additional fields in the metadata.yaml file which i don't think we necessarily need here.